integration_test/ops_agent_test/main_test.go

```diff
@@ -96,17 +95,39 @@ func workDirForImage(imageSpec string) string {
 }
 
 func startCommandForImage(imageSpec string) string {
-	if gce.IsWindows(imageSpec) {
+	isUAPPlugin := agents.IsOpsAgentUAPPlugin()
+
```

Comment:
I think we can improve the if structure here by nesting UAP cases together.
if UAP {
...
}

Reply:
Done